doc/source/advanced.rst

```diff
@@ -517,6 +517,27 @@ The ``runtime_env`` is a Python dictionary including one or more of the followin
 
   - Example: ``{"OMP_NUM_THREADS": "32", "TF_WARNINGS": "none"}``
 
+- ``container`` (Dict[str, Any]): A dictionary configuring the options for running
+  Ray worker inside a container. This is an experimental feature. Currently Ray requires
```

Comment:
```suggestion
  a Ray worker inside a container. This is an experimental feature. Currently Ray requires the
```

Reply:
Can you make the experimental tag more prominent? e.g., [EXPERIMENTAL] A dictionary...